Sources/MapboxNavigation/CameraView.swift

```diff
@@ -0,0 +1,162 @@
+import UIKit
+import MapboxMaps
+
+/**
+ A view that represents a camera view port.
+ */
+public class CameraView: UIView {
```

Comment:
`CameraView` was added temporarily to execute animations using `UIViewPropertyAnimator`.